Examples/Swift/SwitchStylesExample.swift

```diff
@@ -0,0 +1,69 @@
+import UIKit
+import Mapbox
+
+@objc(SwitchStylesExample_Swift)
+
+class SwitchStylesExample: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource, MGLMapViewDelegate {
+    
+    let mapView = MGLMapView()
+    let stylePicker = UIPickerView()
+    var button = UIButton()
+    
+    let styles = [
+        ("Mapbox Light", MGLStyle.lightStyleURL()),
+        ("Mapbox Streets", MGLStyle.streetsStyleURL()),
+        ("Mapbox Dark", MGLStyle.darkStyleURL()),
+        ("Mapbox Outdoors", MGLStyle.outdoorsStyleURL()),
+        ("Mapbox Satellite", MGLStyle.satelliteStyleURL()),
```

Comment:
Tuples are incompatible with Objective-C; in the Objective-C version, maintain an array of style URLs (since the order matters) alongside a single dictionary that maps style URLs to style names.